test/integration/clitest/test_render.py

```diff
@@ -195,13 +195,13 @@ def test_edit(self, target_name, tmpdir):
                 self.assert_channel_rdef(channels[c], rd['channels'][c + 1])
             self.assert_image_rmodel(img, expected_greyscale)
             # img._closeRE()
-        # assert not gw._assert_unregistered("testEdit")
+        # assert not gw._assert_unregistered("testSet")
```

Comment:
you should not be able to test via travis. The feature was included in 5.4.2

Reply:
i.e. you can try to uncomment. If anyone tries to run the tests again an older version, they'll have issues.